frontend/src/components/Search.js

```diff
@@ -0,0 +1,121 @@
+/* eslint-disable no-nested-ternary */
+import React, { useEffect, useState } from "react";
+import Modal from "react-modal";
+import "../styles/Search.css";
+
+const allFiles = [
+  {
+    id: "file_1",
+    name: "MAW File",
+  },
+  {
+    id: "file_2",
+    name: "TSE File",
+  },
+  {
+    id: "file_3",
+    name: "Testing",
+  },
+];
+
+function FileResult({ val, index }) {
```

Comment:
This looks pretty similar to the other file component, should we abstract it into a separate component and use it in the modal and on the file pages?

Reply:
I tried it, you can pretty much drop the `<FileEntry>` tag from FileEntry.js in place of the `<FileResult>` tag and have it work -- the only thing that needs changing (other than renaming the props to match) is the width of the modal content.